### PR TITLE
FF-491 relative timestamps

### PR DESCRIPTION
**Pull request by qvalentin**

### CODE REVIEW

File: src/background/methods/dataTypes/time.ts [hunk 1]
```diff
@@ -6,7 +6,42 @@ function getTimeStampFromDate(date: Date): number {
...
+            const years = Math.round(diff / year);
+            return `${years} year${years > 1 ? "s" : ""} ago`;*/
+        default:
+            return "";
+    }
+}
+
 function getDateAsStringFromTimestamp(ts: number): string {
+    const relativeTime = getRelativeTime(ts);
+    if (relativeTime) {
```
Comment (open-schnick) [inline src/background/methods/dataTypes/time.ts:41]: Does empty string evaluate to _false_?

**Review by open-schnick — CHANGES_REQUESTED** [review]
Check comment

File: src/background/methods/dataTypes/time.ts (hunk 1 above)
Comment (qvalentin) [reply src/background/methods/dataTypes/time.ts:41]: Yes.

Comment (open-schnick) [reply src/background/methods/dataTypes/time.ts:41]: then merge it

**Review by open-schnick — APPROVED** [review]

**Pull request merged**